cocos/deprecated/CCDeprecated.h

```diff
@@ -32,8 +32,59 @@
 #define __COCOS2D_CCDEPRECATED_H__
 
 #include <math.h>
-#include "math/CCGeometry.h"
+#include "deprecated/CCSet.h"
```

Comment:
`CCDeprecated.h` uses lots of deprecated class. Just include `CCDeprecated.h` is not enough if I don't add these includes.